### ISSUE TITLE
How do I actually reset mission progress on a locally hosted mission

### ISSUE THREAD

**Issue by Xeno69** [Xeno69/Domination#491]
Quick question regarding the mission progress saving/loading:
Every time I load any of the various domination missions on a given map (eg. carrier or airbase Altis) the previous progress seems to be automatically loaded. For example, if I capture a main target in the Altis Blufor mission, then load into either that same mission or even if I load into the Carrier version, that main target remains captured along with my non-zero player score. I'm playing locally hosted (not dedi) so this can't be a database related thing.
TLDR: How do I actually reset mission progress on a locally hosted mission?

**Comment by longtimegamer** [comment]
`d_save_to_mpns = 0`

I am not sure where to delete the namespace file, however.  @Xeno69 do you know where it is stored?

**Comment by Xeno69** [comment]
Oh, sorry to respond so late, it is either in your Documents/Arma 3 folder or Documents/Arma 3 - Other Profiles/<player name> folder

**Comment by longtimegamer** [comment]
@Xeno69 Maybe we can fix this?  I was thinking "reset progress" button on the administrator user interface with user confirmation to avoid an accident.

**Comment by longtimegamer** [comment]
workaround:

1) stop the server
2) delete file Documents/Arma 3/dominationmissiongroup.vars
